### ISSUE TITLE
How to use the package with one dimensional data?

### ISSUE THREAD

**Issue by hmars-t** [keisen/tf-keras-vis#86]
Is it possible to apply the method of activation maximization also on a 1D-CNN with one dimensional data?

**Comment by keisen** [comment]
Yes, of course! However, you have to adjust some parameters for one dimensional data, because, in default, the parameters of ActivationMaximization is tuned for 2D-CNN model. 

1.  Modify `input_range`, `input_modifiers` and `regularizers` for your data properly.
2. Tune parameters (`optimizer`, `input_modifiers`, `regularizers`, or so on) to generagte the model inputs that maximize the output of the given score functions.

Please see below for details:

* [tf_keras_vis.activation_maximization.ActivationMaximization.__call__()](https://keisen.github.io/tf-keras-vis-docs/api/tf_keras_vis.activation_maximization.html#tf_keras_vis.activation_maximization.ActivationMaximization.__call__)
* [example for visualize_dense_layer](https://keisen.github.io/tf-keras-vis-docs/examples/visualize_dense_layer.html)


Please let's me know if you face any error.
Thanks!
